packages/components/src/atoms/Link/Link.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { forwardRef } from 'react'
+import type { ElementType, ReactElement } from 'react'
+import { PolymorphicComponentPropsWithRef, PolymorphicRef } from '../../typings'
+
+type Variant = 'default' | 'display' | 'footer' | 'inline'
```

Comment:
@ArthurTriis1 let's change this a little bit. I think `footer` looks strange here.

We can remove `footer` option of type `Variant` and create a new type `Size = 'small' | 'regular'`. And then replace `footer` occurrences for { variant="display", size="small"}. WDYT?